airbyte-config/init/src/main/resources/seed/destination_definitions.yaml

```diff
@@ -191,7 +191,7 @@
 - name: Redshift
   destinationDefinitionId: f7a7d195-377f-cf5b-70a5-be6b819019dc
   dockerRepository: airbyte/destination-redshift
-  dockerImageTag: 0.3.28
+  dockerImageTag: 0.3.30
```

Comment:
I dont think you will be allowed to publish a 0.3.30 again cause an image with this version already exists

Reply:
bumped